client/src/components/PitchDoc/Sidebar.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from 'react';
+
+import Home from '../../assets/home.svg';
+import Person from '../../assets/person.svg';
+import Pitch from '../../assets/pitch.svg';
+import Library from '../../assets/library.svg';
+import People from '../../assets/people.svg';
+
+import '../../css/Sidebar.css';
+
+// Temporary profile pic of Mustafa
+const profilePicPath =
+  'https://ca.slack-edge.com/T6VL1BSEA-U01M1TTQ1TQ-09478f1b309f-512';
+
+function Sidebar(): ReactElement {
+  return (
+    <div className="sidebar">
+      <div className="profile-pic">
+        <img src={profilePicPath} alt="IMG-5592" />
```

Comment:
Semantic does have their own Image tag which has some more functionality, if we want to switch to that to be more consistent across the project